src/dependenpy/cli.py

```diff
@@ -179,6 +179,9 @@ def _run(opts, dsm):
             dsm.print(format=opts.format, output=output, indent=indent)
         elif opts.matrix:
             dsm.print_matrix(format=opts.format, output=output, depth=depth, indent=indent, zero=opts.zero)
+            algorithms.DsmMatrix.create(dsm.as_matrix()).levelize().print(
+                format=opts.format, output=output, depth=depth, indent=indent, zero=opts.zero
+            )
```

Comment:
Instead of printing both matrix, do you think it would give better UX if we added an option to sequence the data/matrix before printing it? So that only one matrix is printed? I'm thinking about big matrices: one already takes a lot of space, so two might be too much. Just wondering.